src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vedtak/vedtaksperiode/VedtakPeriodeValidering.kt

```diff
@@ -0,0 +1,76 @@
+package no.nav.familie.ks.sak.kjerne.behandling.steg.vedtak.vedtaksperiode
+
+import no.nav.familie.ks.sak.common.util.tilKortString
+import no.nav.familie.ks.sak.kjerne.behandling.steg.vedtak.VedtakBegrunnelseType
+import no.nav.familie.ks.sak.kjerne.behandling.steg.vedtak.vedtaksperiode.domene.UtvidetVedtaksperiodeMedBegrunnelser
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+// Håpet er at denne skal kaste feil på sikt, men enn så lenge blir det for strengt. Logger for å se behovet.
+fun List<UtvidetVedtaksperiodeMedBegrunnelser>.validerPerioderInneholderBegrunnelser(
```

Comment:
Denne er ikke ibruk ser det ut til? Vi trenger den kanskje senere?

Reply:
Stemmer, vi trenger den for foreslå og beslutt vedtak. Egentlig denne som startet hele inntrekningen av vedtaksperiode 😆